do_checks.sh

```diff
@@ -28,7 +28,6 @@ cargo clippy --all-features --workspace --all-targets -- \
 cargo clippy --all-features --workspace --lib --bins --examples -- \
     -A clippy::all \
     -D clippy::float_arithmetic \
-    -D clippy::unwrap_used \
```

Comment:
I have not yet found a way to disable the errors clippy is reporting for the uses of unwrap in the generated trezor messages code.

Reply:
What if we fork trezor repo instead? Then we create a branch with mintlayer modifications and point from mintlayer-core.
This would prevent us from adding 100k lines of foreign code, solve clippy problem and keep our modifications isolated.

Reply:
one more reason would be that .proto file is currently missing

Reply:
oh we've already forked that repo, than I'm confused why all the trezor code here?

Reply:
Yes we have the repo forked, I use it locally like that you can see the commented code in one cargo file. The copy was Sam's suggestion, but not sure why.

Reply:
this is reenabled now